tests/core/tools/manual/find.html

```diff
@@ -0,0 +1,27 @@
+<style>
```

Comment:
This is purely API addition and should not be tested manually, as it's possible to test it 100% using automatic testing.